.test-env

```diff
@@ -0,0 +1,8 @@
+# Configs for testing repo download:
+SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
+SDK_TESTING_BRANCH="trim-indexer"
+SDK_TESTING_HARNESS="test-harness"
```

Comment:
Approving with the provision that this PR should wait on the SDK-testing `trim-indexer` branch being merged and then be pointed to `master`.